test_engine/build_test_engine.py

```diff
@@ -0,0 +1,243 @@
+import paramiko
+import subprocess
+import yaml
+import os
+import pathlib
+import json
+import time
+
+docker_id = os.getenv("docker_id")
+docker_pwd = os.getenv("docker_pwd")
+docker_repo = os.getenv("docker_repo")
+
+RETRY_INTERVAL = 0.5
+
+os.environ['TF_VAR_tf_workspace'] \
+    = str(pathlib.Path(__file__).parent.absolute())
+
+# TF_VAR_build_engine_aws_access_key
+# TF_VAR_build_engine_aws_secret_key
+
+
+def get_image_names():
+
+    global CLIAPIVersion
+    global CLIAPIMinVersion
+    global ControllerAPIVersion
+    global ControllerAPIMinVersion
+    global DataFormatVersion
+    global DataFormatMinVersion
+    global docker_tag
+
+    version = subprocess.check_output("{}/scripts/get_version.sh".format(
```

Comment:
where `get_version.sh` is located/generated ?

Reply:
The file located in longhorn-tests/test_framework/scripts/get_version.sh